src/components/TimeButton/TimeButton.stories.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Provider } from 'react-redux';
+import { configureStore, combineReducers, createSlice } from '@reduxjs/toolkit';
+import TimeButton from './TimeButton';
+
+const timeReducer = createSlice({
+  name: 'time',
+  initialState: {
+    time: 45,
+  },
+  reducers: {
+    setTime(state, { payload: { time } }) {
+      return {
+        ...state,
+        time,
+      };
+    },
+  },
+});
+
+const reducers = combineReducers({
+  time: timeReducer.reducer,
+});
+
+const store = configureStore({ reducer: reducers });
+
+const withReduxMockStore = (story) => (
+  <Provider store={store}>{story()}</Provider>
+);
+
+export default {
+  title: 'timeButton',
+  component: TimeButton,
+  decorators: [withReduxMockStore],
+};
+
+const nomal = (args) => <TimeButton {...args} />;
+const clicked = (args) => <TimeButton {...args} />;
+
+export const timeButton = nomal.bind({});
+
+export const clickedTimeButton = clicked.bind({});
+
+timeButton.args = { time: 30 };
+clickedTimeButton.args = { time: 45 };
```

Comment:
아래와 같이 normal 하나로 두고 bind 해주는 것은 어떤가요?

```suggestion
const normal = (args) => <TimeButton {...args} />;

export const timeButton = normal.bind({});
export const clickedTimeButton = normal.bind({});

timeButton.args = { time: 30 };
clickedTimeButton.args = { time: 45 };
```

Reply:
이게 기능을 달리하려고 나눠놨었는데 미쳐 수정 못했네요. 감사합니다!